src/main/java/com/jalasoft/jfc/model/PdfConverter.java

```diff
@@ -61,7 +61,7 @@ public Boolean convert(PdfParam pdf){
                 renderer = new PDFRenderer(documenttoimage);
             }
             for (int i = 0; i < pages; i++) {
```

Comment:
improve the for iterator varible to name more specific, for example **item**

Reply:
ok, it is already fixed